app/helpers/application_helper.rb

```diff
@@ -206,7 +206,7 @@ def searchable?
 
   def auto_complete_search(name, val, options = {})
     path = options[:path] || send("#{controller_name}_path")
```

Comment:
Well, it doesn't need to go back verbatim, it would need to also append the former 'auto_complete_search' onto it as well.